icx-cert/src/main.rs

```diff
@@ -0,0 +1,21 @@
+use anyhow::Result;
+use clap::{crate_authors, crate_version, Clap};
+
+mod pprint;
+
+#[derive(Clap)]
+#[clap(
+    version = crate_version!(),
+    author = crate_authors!(),
+)]
+enum Command {
+    /// Fetches the specified URL and pretty-prints the certificate.
+    #[clap(name = "pprint")]
```

Comment:
Non blocking: Just `print` or even no subcommand at all.

Reply:
Renamed the command to "print".